packages/app/src/Pages/Login.tsx

```diff
@@ -279,12 +294,28 @@ export default function LoginPage() {
           <div className="flex">
             <input
               dir="auto"
-              type="text"
+              type={isMasking ? "password" : "text"}
               placeholder={formatMessage(messages.KeyPlaceholder)}
               className="f-grow"
               onChange={e => setKey(e.target.value)}
             />
+            <Icon
+              name={isMasking ? "openeye" : "closedeye"}
+              size={30}
+              className="highlight btn-sm pointer"
+              onClick={flipMask}
+            />
           </div>
+          <p className="login-note">
+            {suggestMasking && !isMasking && (
+              <span className="highlight">
+                <FormattedMessage
+                  defaultMessage="Please mask your input by clicking the eye icon to protect your privacy"
```

Comment:
I think people know what the eye is for, maybe this message is not needed

Reply:
removed this message now